### PR TITLE
Prepare version 2.6

### PR DESCRIPTION
**Pull request by leplatrem**
**Tooling requirements**
* ``pip install zest.releaser[recommended]`` (Install zest.releaser, wheel and twine)

**Step 1**
```
$ git co -b prepare-X.Y.Z
$ prerelease
$ vim docs/conf.py
$ rm -fr .venv/
$ make install
$ pip freeze > requirements.txt
$ # Remove the master URL in the requirements.txt file.
$ git ci -a --amend
$ git push origin prepare-X.Y.Z
```
* [x] Update Contributors file
* [x] Merge remaining pull requests
* [x] Update changelog
* [x] Update version in docs/conf.py
* [x]  Update known good set of dependencies in requirements.txt

**Step 2**
```
$ git co master
$ git merge --no-ff prepare-X.Y.Z
$ release
$ postrelease
```
* [ ] Tag vX.Y.Z
* [ ] Publish on Pypi

**Step 3**
* [ ] Close milestone in Github
* [ ] Create next milestone in Github
* [ ] Add entry in Github release page
* [ ] Configure the version in ReadTheDocs
* [ ] Send mail to ML

### CODE REVIEW

**Comment by ametaireau** [comment]
related: http://cliquet.readthedocs.org/en/latest/contributing.html#how-to-release

File: cliquet_docs/contributing.rst [hunk 1]
```diff
@@ -129,37 +129,37 @@ Install `zest.releaser` with the `recommended` dependencies. They contain
...
 
+- Open a pull-request with to release the version.
 
 Step 2
 ------
 
-Once the pull request is validated, you can merge it and do a release. We are
-using the `release` command to invoke the `setup.py` builds and upload to PyPI.
+Once the pull-request is validated, merge it and do a release.
+Use using the ``release`` command to invoke the ``setup.py`` builds and upload to PyPI.
```
Comment (ametaireau) [inline cliquet_docs/contributing.rst]: Use using

Comment (ametaireau) [inline cliquet_docs/contributing.rst]: Should be "Use the release command to invoke the setup.py, which builds and uploads to PyPI".

File: cliquet_docs/contributing.rst [hunk 2]
```diff
@@ -129,37 +129,37 @@ Install `zest.releaser` with the `recommended` dependencies. They contain
...
 
 .. code-block:: bash
 
     $ git checkout master
-    $ git merge --no-ff prepare-X.X
+    $ git merge --no-ff prepare-X.Y.Z
+    $ git tag X.Y.Z
```
Comment (ametaireau) [inline cliquet_docs/contributing.rst]: I thought the tag was done with the postrelease command?

Comment (Natim) [inline cliquet_docs/contributing.rst]: In the release command yes.

File: cliquet/tests/resource/test_object_permissions.py [hunk 3]
```diff
@@ -114,9 +114,9 @@ def test_permissions_are_modified_with_patch(self):
         self.resource.request.validated = {'permissions': perms}
         self.resource.request.method = 'PATCH'
         result = self.resource.patch()
-        self.assertDictEqual(result['permissions'],
-                             {'write': ['basicauth:userid', 'jean-louis'],
-                              'read': ['fxa:user']})
+        self.assertEqual(result['read']['read'], ['fxa:user']),
```
Comment (Natim) [inline cliquet/tests/resource/test_object_permissions.py]: should be ['permission']['read']

**Pull request merged**